### DISCUSSION TITLE
Guess the number

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#61] [Galaxy Qs] [answered]
- The first digit is a prime number.
- The second digit is the fifth digit minus the first digit.
- The third digit is twice the first digit.
- The fourth digit is the third digit plus three.
- The fifth digit is the difference between the first digit and the fourth digit.

**Comment by LucaConstantin** [comment]


**Reply by Sriansh-raj** [reply]
There is

and that's why I've given such qs

**Reply by Sriansh-raj** [reply]
What about **23475**

**Comment by mamadjavaad** [comment, marked as answer]
33696 
1- The first digit of 33696 is 3, which is a prime number, so this clue is satisfied.
2- The second digit of 33696 is 3 and the fifth digit is 6. Since 6 minus 3 is equal to 3, this clue is satisfied.
3- The third digit of 33696 is 6, which is equal to twice the first digit (3), so this clue is satisfied.
3- The fourth digit of 33696 is 9, which is equal to the third digit (6) plus three, so this clue is satisfied.
4- The fifth digit of 33696 is 6 and the difference between the first (3) and fourth (9) digits is also 6, so this clue is satisfied too so 33696 is correct answer.

**Reply by Sriansh-raj** [reply]
Yeah another answer 👍🏻 
Goodjob :)

**Comment by Denisyasyuchenya** [comment]
33696
